addons/medical_statemachine/functions/fnc_enteredStateDeath.sqf

```diff
@@ -16,13 +16,14 @@
  */
 
 params ["_unit"];
-if (isNull _unit) exitWith {};
+if (isNull _unit || {!isNil {_unit getVariable QEGVAR(medical,causeOfDeath)}}) exitWith {
+    WARNING_1("enteredStateDeath: State transition on dead or null unit - %1",_unit);
```

Comment:
```suggestion
    if ((_unit getVariable [QEGVAR(medical,causeOfDeath), ""]) == "#scripted") exitWith {};
    WARNING_1("enteredStateDeath: State transition on dead or null unit - %1",_unit);
```
We don't want to show warning for scripted deaths (`cursorObject setDammage 1`)
Because of the manual transition setDead will run before this and have already set the var